.circleci/config.yml

```diff
@@ -154,7 +154,29 @@ workflows:
           working_directory: sites/partners
           yarn: true
           build: |
+            echo 'export FEATURE_LISTINGS_APPROVAL=FALSE' >> "$BASH_ENV"
+            source "$BASH_ENV"
             yarn test:backend:core:dbsetup
-          start: yarn dev:all-cypress
+          start: |
+            yarn dev:all-cypress
+          command: |
+            npx cypress run --spec cypress/e2e/default/**/*.{js,jsx,ts,tsx}
+          wait-on: "http://0.0.0.0:3001"
+          store_artifacts: true
+      - cypress/run:
```

Comment:
This is a bummer that we have to add a full new suite, but I understand why we need that since it's a build env that we are changing.

Reply:
I'm wondering what our long term plan for this should be. For every new feature that we want to permanently be around a feature flag it's probably not sustainable to create a new test suite. Theoretically moving feature flags to the new system (new feature flag table) should fix this since we can just set the flag at the jurisdiction level in the DB. So as I typed this out I think I've answered my own question